tests/e2e/fixtures/EUR.js

```diff
@@ -1,17 +1,24 @@
-import checkout from "../pages/CheckoutPage";
-import NL from "./countriesEUR/NL";
+import CheckoutPage from "../pages/CheckoutPage";
+import testNetherlands from "./countriesEUR/NL";
+import testFrance from "./countriesEUR/FR";
+import testGermany from "./countriesEUR/DE";
+import testAustria from "./countriesEUR/AT";
 import { regionsEnum } from "../data/enums"
-//TODO check if we want to move URL
+
 fixture`EUR`
     .page(`https://${process.env.SFCC_HOSTNAME}/s/RefArch/home?lang=${regionsEnum.EU}`)
```

Comment:
region does not need to be in this URL as we change locales during checkout

Reply:
Yes, makes sense. Will apply it.